allure-report-face/src/blocks/widget/styles.css

```diff
@@ -36,11 +36,20 @@
   &__handle {
     display: none;
     position: absolute;
-    font-size: 16px;
-    cursor: move;
     right: $gap-size;
     top: $gap-size;
     color: $text-muted-color;
+    cursor: move; /* fallback if grab cursor is unsupported */
+    cursor: grab;
+    cursor: -moz-grab;
+    cursor: -webkit-grab;
+
+    &:active {
+      cursor: grabbing;
+      cursor: -moz-grabbing;
+      cursor: -webkit-grabbing;
+    }
```

Comment:
:+1: for the cursor thing.
Don't add prefixes. We have autoprefixer, that takes care about that.

:-1: for changing the icon. You can extend draggable area around the icon. It probably will help better that shedding around icon look.